app/views/schools/school_targets/_limited_data.html.erb

```diff
@@ -4,13 +4,13 @@
   </div>
   <div class="col-md-8 align-self-center work_week d-none d-sm-block" style="padding-right: 25px">
     <div class="d-flex justify-content-center mt-2">
-      <span class="text-center">
+      <h4 class="text-center">
         <% if overview_data.present? && overview_data.work_week(fuel_type).has_data %>
-              <%= up_downify(overview_data.work_week(fuel_type).change) %> last week against target of <%= target %>%
+              <%= up_downify(overview_data.work_week(fuel_type).change) %> last week against target of -<%= target %>%
```

Comment:
spacing looks off here by a couple if chars?  (and a nitpick - maybe use `<%=-target%>` as above for consistency?)